### ISSUE TITLE
volume-slider initial value/view is incorrect

### ISSUE THREAD

**Issue by tomByrer** [muxinc/media-chrome#197]
I found your top-secret examples folder! 🕵️

I first opened the [Wistia player](https://media-chrome.mux.dev/examples/media-elements/wistia.html), & the volume visually was set to 0, but I could still hear audio:
![image](https://user-images.githubusercontent.com/1308419/161436244-16e4d632-d6b9-44e6-9b8e-c7b631282ba6.png)

Note, I never visited this page before; new private window also sets to 0 visually, but not new tab in same private window.
Manually adjusting volume gives expected results.  Subsequent page openings retained my volume setting,.

_Preferred fix_: Default audio to be set to half (0.5 | 50) both audibly & visually.

**Comment by tomByrer** [comment]
Also, note how the volume-slider is set to 0, but the mute-button is not muted.

Noticed it more clearly in [this (beta) example](https://media-chrome-git-fork-cjpillsbury-improv-icons-cleanup-mux.vercel.app/examples/themes/netflix-theme.html), where default actual audio & slider levels match (no sound), but it is not 'muted':
![image](https://user-images.githubusercontent.com/1308419/161437649-e137d567-9375-42ff-9102-2f3cf2095b27.png)

**Comment by dylanjha** [comment]
@tomByrer thank you for bringing this up! (the examples folder isn't _meant_ to be top secret, so I'm glad you found it, but also making we think we should surface that a little better :).

There is this "feature" to remember the volume setting across page refreshes, so that part is at least intended.

https://github.com/muxinc/media-chrome/blob/52ccddc6a7fd7fbe8322da5e59f26667ce0a5509/src/js/media-controller.js#L107-L114

I tried out the Wistia player, I'm actually not able to reproduce the bug that you're seeing (am I missing something?):

https://user-images.githubusercontent.com/764988/161606664-803ee839-0a83-4c8b-9a39-5024f2f9e1d3.mov

**Comment by tomByrer** [comment]
> tried out the Wistia player, I'm actually not able to reproduce the bug

Opened in a private window, or cleared all cache/cookies/localstorage?
Remember, I discovered the 0 volume in 2 unvisted browsers.

**Comment by dylanjha** [comment]
@tomByrer good call! I am able to repro in incognito. It does not happen with the other examples I tested.

There might be 2 things here:

- A bug with Wistia example that's causing the volume to start at 0
- A possible improvement to default the volume to 50%

Does that sound right? @luwes could you weight in here?

**Comment by gkatsev** [comment]
I would find it weird if I set up media-chrome and the volume defaults to 50%.

**Comment by luwes** [comment]
ah yes I can reproduce as well, that looks like a bug in https://github.com/luwes/wistia-video-element 
I'll have a look soon

**Comment by tomByrer** [comment]
> the native video default volume is 1 or 100%, best to keep that.

Well, that is better than visual 0% :)

Note: there is are use-cases for default 0%:
- not embarrass yourself & irritate others in public spaces ;)
- IIRC web browsers won't allow auto-play for videos if audio is on

**Comment by luwes** [comment]
Closing as these are working as expected now
